ssg/utils.py

```diff
@@ -90,6 +90,10 @@ def parse_name(product):
     if match:
         _product = match.group(1)
         _product_version = match.group(2)
+        # Ubuntu versions have a dot in between the numbers
+        # While the prodtype doesn't have the dot, the full product name does
+        if _product == "ubuntu":
+            _product_version = _product_version[:2] + "." + _product_version[2:]
```

Comment:
As `_parse_name()` is only used by `_check_is_applicable_for_product()`, I decided to update Ubuntu's version here.
But design-wise, it may be better to do it after call for `_parse_name()`:
https://github.com/ComplianceAsCode/content/blob/0e5705bedb3ccb3e6840a862598260ce42dae38e/ssg/build_ovals.py#L27

Thoughts?

Reply:
I've decided to move the Ubuntu version handling out of `_parse_name()` and into `_check_is_applicable_for_product()`.

I think it is cleaner this way.